src/Sulu/Bundle/MediaBundle/Tests/Functional/Controller/MediaStreamControllerWebsiteSecuredTest.php

```diff
@@ -0,0 +1,179 @@
+<?php
+
+/*
+ * This file is part of Sulu.
+ *
+ * (c) Sulu GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Bundle\MediaBundle\Tests\Functional\Controller;
+
+use Sulu\Bundle\ContactBundle\Entity\Contact;
+use Sulu\Bundle\MediaBundle\Api\Media;
+use Sulu\Bundle\MediaBundle\Collection\Manager\CollectionManager;
+use Sulu\Bundle\MediaBundle\DataFixtures\ORM\LoadCollectionTypes;
+use Sulu\Bundle\MediaBundle\DataFixtures\ORM\LoadMediaTypes;
+use Sulu\Bundle\MediaBundle\Entity\Collection;
+use Sulu\Bundle\MediaBundle\Media\Manager\MediaManager;
+use Sulu\Bundle\MediaBundle\Tests\Application\SecuredKernel;
+use Sulu\Bundle\SecurityBundle\Entity\User;
+use Sulu\Bundle\SecurityBundle\Entity\UserRole;
+use Sulu\Bundle\TestBundle\Testing\WebsiteTestCase;
+use Symfony\Bundle\FrameworkBundle\KernelBrowser;
+use Symfony\Component\HttpFoundation\File\UploadedFile;
+
+/**
+ * @runTestsInSeparateProcesses else we get a conflict with the WebspaceCollectionClass already exists error.
+ */
+class MediaStreamControllerWebsiteSecuredTest extends WebsiteTestCase
+{
+    /**
+     * @var KernelBrowser
+     */
+    private $client;
+
+    protected static function getKernelClass(): string
+    {
+        return SecuredKernel::class;
+    }
+
+    public function setUp(): void
+    {
+        if (\version_compare(SecuredKernel::VERSION, '5.4', '<')) { // @phpstan-ignore-line
+            $this->markTestSkipped('Test is written for Symfony 5.4 or newer.');
+        }
```

Comment:
The test does not work on lowest because we need `enable_authenticator_manager`. In Sulu 2.5 this line can be removed when merging back.